### PR TITLE
[4.9] Align code formatting

### PR DESCRIPTION
**Pull request by Donnerbart**
https://hivemq.kanbanize.com/ctrl_board/22/cards/6931/details/

### CODE REVIEW

File: src/main/java/com/hivemq/extension/sdk/api/auth/parameter/EnhancedAuthOutput.java [hunk 1]
```diff
@@ -127,10 +130,12 @@ public interface EnhancedAuthOutput extends AsyncOutput<EnhancedAuthOutput> {
     /**
      * Successfully authenticates the client.
      * <p>
-     * During authentication a CONNACK packet with reason code {@link com.hivemq.extension.sdk.api.packets.connect.ConnackReasonCode#SUCCESS
+     * During authentication a CONNACK packet with reason code
+     * {@link com.hivemq.extension.sdk.api.packets.connect.ConnackReasonCode#SUCCESS
      * SUCCESS} and no authentication data is sent to the client.
```
Comment (SgtSilvio) [inline src/main/java/com/hivemq/extension/sdk/api/auth/parameter/EnhancedAuthOutput.java:135]: why is this in new line?

File: src/main/java/com/hivemq/extension/sdk/api/services/cluster/ClusterDiscoveryCallback.java [hunk 2]
```diff
@@ -33,9 +33,11 @@
 public interface ClusterDiscoveryCallback {
 
     /**
-     * This method is called once by HiveMQ when this callback is added via {@link ClusterService#addDiscoveryCallback(ClusterDiscoveryCallback)}.
+     * This method is called once by HiveMQ when this callback is added via
+     * {@link ClusterService#addDiscoveryCallback(ClusterDiscoveryCallback)}.
      * <p>
-     * It can be used to register this HiveMQ instance with some kind of central registry or to save it to a database or
+     * It can be used to register this HiveMQ instance with some kind of central registry or to save it to a database
+     * or
```
Comment (SgtSilvio) [inline src/main/java/com/hivemq/extension/sdk/api/services/cluster/ClusterDiscoveryCallback.java:40]: weird

File: src/main/java/com/hivemq/extension/sdk/api/auth/parameter/EnhancedAuthOutput.java (hunk 1 above)
Comment (Donnerbart) [reply src/main/java/com/hivemq/extension/sdk/api/auth/parameter/EnhancedAuthOutput.java:135]: `com.hivemq.extension.sdk.api.packets.connect.ConnackReasonCode#SUCCESS` can't be wrapped and exceeds the 120 limit, so it moves the whole link into a new line. Makes sense to me.

Comment (Donnerbart) [reply src/main/java/com/hivemq/extension/sdk/api/auth/parameter/EnhancedAuthOutput.java:135]: I guess the formatter is again too stupid to reformat the whole paragraph in a more senseful way, e.g.:
```
     * During authentication a CONNACK packet with reason code {@link
     * com.hivemq.extension.sdk.api.packets.connect.ConnackReasonCode#SUCCESS SUCCESS} and no authentication data is
     * sent to the client.
```

File: src/main/java/com/hivemq/extension/sdk/api/services/cluster/ClusterDiscoveryCallback.java (hunk 2 above)
Comment (Donnerbart) [reply src/main/java/com/hivemq/extension/sdk/api/services/cluster/ClusterDiscoveryCallback.java]: I've rolled back this change (I anyway had to rebase the backports due to the version bump)...

**Review by SgtSilvio — APPROVED** [review]

**Review by SgtSilvio — APPROVED** [review]

**Pull request merged**